### DISCUSSION TITLE
Sometimes the explorer selected item moves to the next item when I press space

### DISCUSSION THREAD

**Discussion by Giresharu** [QL-Win/QuickLook#1449] [Q&A] [answered]
This resulted in sometimes the items I previewed becoming unintended items.
After my attempts, I found that this is related to the length of time the space is pressed, but sometimes I feel that I release it very quickly, which may still trigger this problem.
And if you keep pressing the space button, the selected items may not necessarily cycle through all files in order, but sometimes just jump among individual files.

I'm running Win11 and I'm not sure if this is a problem with Explorer itself, but if it is, how do you resolve keystroke conflicts like this?

https://github.com/user-attachments/assets/9bf23730-1b63-4ad3-b1f0-2a3bf2531ceb

**Comment by Giresharu** [comment]
I really can't think of whether it is affected by third-party software. Currently, I have tested closing all AHK codes, Listary, PowerToy and other software that may be related to the resource manager and keyboard shortcut, and all have this problem. The only thing that is not turned off is QTTabBar, but I don't know how to conveniently turn off its function for testing without uninstalling it. And now I can't uninstall it.
Can any friend help verify it?

**Comment by rabelux** [comment, marked as answer]
Have a look at #726. It's indeed QTTABBAR.

**Reply by Giresharu** [reply]
> Have a look at #726. It's indeed QTTABBAR.

Thank you!
